src/org/jug/bodensee/UCIController.java

```diff
@@ -81,7 +85,7 @@ public void go() {
         send("go inifinite");
     }
     
-    private void send(String s) {
+    public void send(String s) {
```

Comment:
No public modifier allowed. This is just an internal method! Please fix to be merged.